recipes/rmm/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "rmm" %}
+{% set version = "21.12.00" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rapidsai/rmm/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 71802422a04f2e0a828f7b7ec77e5cd1d24361d6dc5bf60a534659e4c2e08a44
+  folder: rmm
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version == "None") or (aarch64) or (py < 37) or (py >= 39)]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake >=3.18
+    - make
+  host:
+    - librmm ={{ version }}
+    - python
+    - pip
+    - numba >=0.49
+    - numpy
+    - cython =0.29
```

Let's stick to the upstream as much as we can (ex: numba is not a build time dependency):
```suggestion
    - librmm {{ version }}
    - python
    - pip
    - setuptools
    - cython >=0.29,<0.30
    - spdlog>=1.8.5,<2
```

---

Done.